pkgs/by-name/ed/edcas-client/package.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, cmake
+, wayland
+, libX11, libXcursor, libXi, libXrandr
+, libxkbcommon, libxcb
+, libglvnd
+}:
+
+let
+  rpathLibs = [
+    libglvnd
+    libXcursor
+    libXi
+    libxkbcommon
+    libXrandr
+    libX11
+    wayland
+  ];
+in rustPlatform.buildRustPackage rec {
+  pname = "edcas-client";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "arne-fuchs";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-1IkaulUeTdAbLSaVmINaAMrfAK5SSGFpW2l04PPZylA=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
```

Comment:
Was reading the docs on this, isn't this only done when building a non-Rust package including Rust code?
Ref: https://nixos.org/manual/nixpkgs/stable/#compiling-rust-applications-with-cargo